plugins/Annotations/API.php

```diff
@@ -225,6 +263,7 @@ public function getAll(string $idSite, ?string $date = null, string $period = 'd
      *                 ),
      *                 ...
      *               )
+     * @throws Exception when user is not a superuser
```

Comment:
Misplaced comment because it should not do that for `getAll` and `getAnnotationCountForDates`?

Reply:
Updated